packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -192,10 +192,13 @@ const getSubmitText = (platform: string, is_eu: boolean, needs_poi: boolean, typ
 
 const IconType = React.memo(({ platform, type, is_eu }: TIconTypeProps) => {
     if (platform === CFD_PLATFORMS.DXTRADE) {
-        if (type === 'synthetic') {
-            return <Icon icon='IcDxtradeSyntheticPlatform' size={128} />;
-        } else if (type === 'financial') {
-            return <Icon icon='IcDxtradeFinancialPlatform' size={128} />;
+        switch (type) {
+            case 'synthetic':
+                return <Icon icon='IcDxtradeSyntheticPlatform' size={128} />;
```

Comment:
Since we will have just one option for Dxtrade from now on, we use different icons for backward compatibility ?

Reply:
@hirad-rewok We still use these icons for the real tab in dxtrade dashboard. We will remove these `switch` as soon as we release the real account.

P.S: I added the TODO for this.